client/gutenberg/editor/components/header/mode-switcher/index.js

```diff
@@ -32,29 +33,22 @@ const MODES = [
 ];
 
 function ModeSwitcher( { onSwitch, mode } ) {
-	const choices = MODES.map( ( choice ) => {
+	const choices = MODES.map( choice => {
 		if ( choice.value !== mode ) {
 			return { ...choice, shortcut: shortcuts.toggleEditorMode.label };
 		}
 		return choice;
 	} );
 
 	return (
-		<MenuGroup
-			label={ __( 'Editor' ) }
-			filterName="editPost.MoreMenu.editor"
-		>
-			<MenuItemsChoice
-				choices={ choices }
-				value={ mode }
-				onSelect={ onSwitch }
-			/>
+		<MenuGroup label={ __( 'Editor' ) } filterName="editPost.MoreMenu.editor">
```

Comment:
What does this `filterName` property? I don't see it [on Gutenberg](https://github.com/WordPress/gutenberg/blob/2a66db0fc99a9fb1ba99f61e59e6a0b2a4a5f9ef/packages/edit-post/src/components/header/mode-switcher/index.js#L41).

Reply:
Not sure, it was there before (this change is just a prettification, the only thing that I've actually modified in this file is the shortcuts import path).